src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+#MySQL database credentials
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://:3306/utopia
+spring.datasource.username=
+spring.datasource.password=
+
+spring.mvc.throw-exception-if-no-handler-found=true
+spring.resources.add-mappings=false
+
+spring.jpa.hibernate.ddl-auto = update
+
+spring.jpa.hibernate.naming.implicit-strategy=org.hibernate.boot.model.naming.ImplicitNamingStrategyLegacyJpaImpl
```

Comment:
If fields are named according to the standard Java conventions, we don't need these naming-strategy properties anymore.